Makefile

```diff
@@ -496,6 +496,9 @@ test-debug: test-build
 test-message: test-build
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) message
 
+test-wpt: all
```

Comment:
Don't we need to call this as a part of test?

Reply:
No, it’s just a convenient shortcut to ‘tools/test.py wpt’ since we also have other kinds of shortcuts in the Makefile. test.py runs every test folder by default unless it’s explicitly put in a black list so it’s covered in the CI and make test